### ISSUE TITLE
Binge Eating

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11587]
**Preferred term label:**
Binge Eating

**Synonyms**
Binge eating behavior; Binge consumption; Overeating

**Definition (free text, please give PubMed ID)**
A behavioral abnormality characterized by recurrent, discrete episodes of rapid and excessive food consumption that is out of proportion to metabolic need or typical consumption under similar circumstances, accompanied by a subjective sense of loss of control over eating and significant psychological distress.
PMID: [19501749](https://pubmed.ncbi.nlm.nih.gov/19501749/)

**Parent term (use hpo.jax.org/app)**
HP:0100738/Abnormal eating behavior 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784

**Comment by pnrobinson** [comment]
@SmartMonkey-git Related issue is Bulimia (HP:0100739), which is perhaps similar to Hypomanic episode (bundled). Let's look at the annotations and see if this should be replaced by Binge eating which is closer to an atomic phenotype.

**Comment by SmartMonkey-git** [comment]
@pnrobinson If you plan to replace Bulimia with Binge eating, then I would recommend to add a second term called `Self-Induced Vomiting`, so the profile of Bulimia can be reconstructed.

Preferred term label:
Self-Induced Vomiting

Synonyms
Self-Induced emesis

Definition (free text, please give PubMed ID)
A compensatory behavior involving the intentional provocation of vomiting to reduce consumed calories and achieve weight control. It typically follows binge eating episodes and is commonly induced manually using fingers down the throat or via psychological prompting.
PMID: [35685564](https://pubmed.ncbi.nlm.nih.gov/35685564/)

Parent term (use hpo.jax.org/app)
Vomiting/HP:0002013

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

Your nano-attribution (ORCID)
https://orcid.org/0009-0002-1118-4784
